aziotctl/aziotctl-common/src/config/apply.rs

```diff
@@ -213,6 +213,22 @@ pub fn run(
 
                                 aziotid_keys.keys.push(super::DEVICE_ID_ID.to_owned());
 
+                                // Identity Service needs authorization to manage temporary credentials
+                                // during cert rotation.
+                                if let Some(auto_renew) = &auto_renew {
+                                    let temp = format!("{}-temp", super::DEVICE_ID_ID);
+
+                                    cert_issuance_certs.insert(
```

Comment:
@arsing There's currently no way to specify issuance options in the create_cert API call or tell it to use another cert's issuance options. So we must create a "device-id-temp" cert issuance entry that duplicates the settings for "device-id". Thoughts?